src/components/common/TmSessionWelcome.vue

```diff
@@ -12,6 +12,13 @@
 
       <div class="tm-session-main">
         <LiSession
+          v-if="!browserWithLedgerSupport"
+          icon="usb"
+          title="Sign in with Ledger Nano S"
+          subtitle="Please use Chrome, Opera, or Brave."
```

Comment:
If the LiSession doesn't have a click event  we should remove the `:hover` pointer and background changes. Could confuse the user.

Ideally the 'Please use…` message is highlighted, but perhaps not a deal breaker.

Reply:
@colw I agree with you. I think I'll need to modify `LiSession` to do this but I feel like this should be done as a separate issue.

@jbibla Thoughts?

Reply:
@jbibla I think I'll open a new issue for this since the scope of this issue has expanded.